src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,46 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+  private Set<Integer> numbers;
+
+  @BeforeEach
+  void setUp() {
+    numbers = new HashSet<>();
+    numbers.add(1);
+    numbers.add(1);
+    numbers.add(2);
+    numbers.add(3);
+  }
+
+  @Test
+  @DisplayName("Set의 크기 확인 테스트")
+  void size() {
+    assertThat(numbers.size()).isEqualTo(3);
+  }
+
+  @ParameterizedTest
+  @ValueSource(ints = {1, 2, 3})
+  @DisplayName("Set의 1, 2, 3 값 존재 확인 테스트")
+  void contains(int number) {
+    assertThat(numbers.contains(number)).isTrue();
+  }
+
+  @ParameterizedTest
+  @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
+  @DisplayName("Set의 입력값에 따른 결과값 확인 테스트")
```

Comment:
```suggestion
  @DisplayName("입력 값에 따라 결과 값이 다른 경우에 대한 테스트")
```
- 기능 요구사항을 테스트케이스 DisplayName으로 그대로 활용해주셔도 좋습니다. 저는 현업에서도 기능요구사항 도출 후 TODO로 작성해두고, 이를 테스트 케이스로 활용하는 편입니다.